bindings/python/unicorn/unicorn.py

```diff
@@ -89,8 +89,8 @@ def _setup_prototype(lib, fname, restype, *argtypes):
 _setup_prototype(_uc, "uc_close", ucerr, uc_engine)
 _setup_prototype(_uc, "uc_strerror", ctypes.c_char_p, ucerr)
 _setup_prototype(_uc, "uc_errno", ucerr, uc_engine)
-_setup_prototype(_uc, "uc_reg_read", ucerr, uc_engine, ctypes.c_int, ctypes.c_void_p)
-_setup_prototype(_uc, "uc_reg_write", ucerr, uc_engine, ctypes.c_int, ctypes.c_void_p)
+_setup_prototype(_uc, "uc_reg_read", ucerr, uc_engine, ctypes.c_uint64, ctypes.c_void_p)
```

Comment:
the original prototype of uc_reg_read() is like this:

    uc_err uc_reg_read(uc_engine *uc, int regid, void *value);

so you cannot use c_uint64 here, but just c_uint.